### PR TITLE
[1LP][RFR] Refined library search so that the index will always be zero.

### PR DESCRIPTION
**Pull request by jteehan**
Purpose or Intent
=================

__Updating tests__  because the library indexes changed after I added SCVMM 2016

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 41222bd292219065e7bf5a828887a63e735b9b07:

`scripts/template_upload_scvmm.py`:
- :bangbang: Line 58:101: **E501** *line too long (101 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 98d6259c6ef672acca19ed99cc1438a672c23d58:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: scripts/template_upload_scvmm.py [hunk 1]
```diff
@@ -144,7 +143,7 @@ def run(**kwargs):
                                    kwargs.get('image_url'), kwargs.get('template_name'))
         check_kwargs(**kwargs)
         mgmt_sys = cfme_data['management_systems'][provider]
-        hostname_fqdn = mgmt_sys['hostname']
+        host_fqdn = mgmt_sys['hostname_fqdn']
```
Comment (psav) [inline scripts/template_upload_scvmm.py]: Not merging as yamls currently have ```hostnam_fqdn``` as they key and not ```hostname_fqdn```

**Comment by amavinag** [comment]
@psav updated yaml to fix the type. this PR is ready to merge.

**Pull request merged**